src/client/src/containers/SelectFrameworks/index.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { connect } from "react-redux";
+
+import styles from "./styles.module.css";
+
+import SelectBackEndFramework from "../SelectBackendFramework";
+import SelectFrontEndFramework from "../SelectFrontEndFramework";
+import { getVSCodeApiSelector } from "../../selectors/vscodeApiSelector";
+
+import DependencyInfo from "../../components/DependencyInfo";
+
+import { AppState } from "../../reducers";
+
+interface ISelectFrameworksProps {
+  vscode: any;
+  isPreview: boolean;
+}
+
+type Props = ISelectFrameworksProps;
+
+class SelectFrameworks extends React.Component<Props> {
+  componentDidMount() {
+    const { vscode, isPreview } = this.props;
+    if (isPreview) {
+      // send messages to extension to check dependency info when this component loads
+      vscode.postMessage({
+        module: "DependencyChecker",
```

Comment:
I think you should use `EXTENSION_MODULES` and `EXTENSION_COMMANDS` for consistency

Reply:
good catch